documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+
+The initial focus is to use a (legacy) data source through NLX as source of the information to be issued and a custom made 'paper wallet' solution as verifiable credential platform. The architecture prepares for usage of other platforms instead of this in the future like IRMA and  prepares for integration into automated needs matching services envisioned within Discipl.
+
+This document describes a high level solution architecture with software components as most detailed level and following ISO-42010 and NORA five layer model in a most pragmatic lean way intended to be readable for all stakeholders involved.
+
+Note: expect this document to change as it is work in progress.
+
+## Foundational layer
+
+The solution might need to be compliant with the following laws, architectures and regulations:
+
+- NORA / GEMMA, and all implications from this though noteworthy:
+- EIF/EiDAS
+- GDPR
+- EN 301 549 / WCAG 2.0 (accessibility)
+- ETSI TS 103 171 (XAdES) for paper wallet signatures
+- BIR2017 (security management)
+- REGULATION (EU) No 910/2014 (as far as applicable)
+
+Note that the core waardepapieren solution itself does not implement or include identification / authentication services for citizens
+nor any key distribution mechanism which is more dependent on it's integration and the specific platforms being used.
+
+As for NORA and GEMMA, Discipl aims at a new innovative architecture to be integrated in NORA in the Future, aiming at a society that is facilitated with a hybrid distributed linked data landscape that aligns with concepts like attribute based credentials and self sovereign identity. And something that is called "Common Ground" is a similair innovation program for GEMMA. This is why we take the liberty to aim the solution to adhere to the principles of Discipl and common ground over those articulated in NORA and GEMMA and defer the identification and resolution of possible discrepancies within these principles to the ongoing development of discipl, common ground (including NLX).
+
+As NLX aims at conforming to EIF/EiDAS and by having the waardepapieren solution use this platform for information retrieval and the NLX certificates for signing (and validation) and by having it not include the identification / authentication of actors, this solution quite automaticly adheres to EIF/EiDAS.
```

Comment:
and by having it not include the -> and by externalizing the
Maybe?

automaticly -> automatically

Reply:
changed it